src/charts/sparkline/sparkline-chart.directive.js

```diff
@@ -0,0 +1,327 @@
+             <input type="radio" ng-model="config.tooltipType" value="default">Default</input>
+           </label>
+           <label class="radio-inline">
+             <input type="radio" ng-model="config.tooltipType" value="usagePerDay">Usage Per Day</input>
+           </label>
+           <label class="radio-inline">
+             <input type="radio" ng-model="config.tooltipType" value="valuePerDay">Value Per Day</input>
+           </label>
+           <label class="radio-inline">
+             <input type="radio" ng-model="config.tooltipType" value="percentage">Percentage</input>
+           </label>
+         </div>
+       </form>
+       <div class="row">
+         <div class="col-md-6">
+           <form role="form"">
+             <div class="form-group">
+               <label>Show</label>
+               </br>
+               <label class="checkbox-inline">
+                 <input type="checkbox" ng-model="custShowXAxis">X Axis</input>
+               </label>
+               <label class="checkbox-inline">
+                 <input type="checkbox" ng-model="custShowYAxis">Y Axis</input>
+               </label>
+             </div>
+           </form>
+         </div>
+         <div class="col-md-3">
+           <form role="form" >
+             <div class="form-group">
+               <label>Chart Height</label>
+               </br>
+               <input style="height:25px; width:60px;" type="number" ng-model="custChartHeight"></input>
+             </div>
+           </form>
+         </div>
+         <div class="col-md-3">
+               <button ng-click="addDataPoint()">Add Data Point</button>
+         </div>
+       </div>
+     </div>
+   </div>
+ </file>
+
+ <file name="script.js">
+ function ChartCtrl($scope) {
+
+     $scope.config = {
+       'chartId': 'exampleSparkline',
+       'tooltipType': 'default'
+     };
+
+    var today = new Date();
+    var dates = ['dates'];
+    for (var d = 20 - 1; d >= 0; d--) {
+        dates.push(new Date(today.getTime() - (d * 24 * 60 * 60 * 1000)));
+    }
+
+     $scope.data = {
+         'total': '100',
+         'data': ['used', '10', '20', '30', '20', '30', '10', '14', '20', '25', '68', '54', '56', '78', '56', '67', '88', '76', '65', '87', '76'],
+         'xData': dates
+     };
+
+     $scope.custShowXAxis = false;
+     $scope.custShowYAxis = false;
+     $scope.custChartHeight = 60;
+
+     $scope.addDataPoint = function () {
+       $scope.data.data.push(Math.round(Math.random() * 100));
+       $scope.data.xData.push(new Date($scope.data.xData[$scope.data.xData.length - 1].getTime() + (24 * 60 * 60 * 1000)));
+     };
+   };
+ </file>
+ </example>
+ */
+angular.module('patternfly.charts').directive('pfSparklineChart', ['c3ChartDefaults',
+  function (c3ChartDefaults) {
+    'use strict';
+    return {
+      restrict: 'A',
+      scope: {
+        config: '=',
+        chartData: '=',
+        chartHeight: '@',
+        showXAxis: '@',
+        showYAxis: '@'
```

Comment:
Shouldn't these use `=` instead of `@`?  This is why you have to use ` $scope.showXAxis === 'true'` below.